.gitlab/test.yml

```diff
@@ -0,0 +1,33 @@
+test:
+  extends: .check_skip_rust
+  timeout: 2 hours
+  image: paritytech/ci-unified:bullseye-1.77.0
+  stage: test
+  cache:
+    key: cache-test
```

Comment:
Can we make the key different as for the other step below? So they would not interfere with each other.

Reply:
done